release-notes/aws/v10.1.0.md

```diff
@@ -0,0 +1,37 @@
+# :zap: Giant Swarm Release 10.1.0 for AWS is now active for you! :zap:
+
+We are pleased to announce that the new Cluster API enabled release v10.1.0 contains the much anticipated Node Pools feature.
```

Comment:
```suggestion
We are pleased to announce that the new release v10.1.0 contains the much anticipated Node Pools feature.
```

Reply:
Cluster API is so far irrelevant for the users. Happy to have this in the changelog. But only confuses here imo.